UnityProject/Assets/Scripts/Health/Objects/Armor.cs

```diff
@@ -138,6 +147,59 @@ public float GetRating(AttackType attackType)
 
 		return 0;
 	}
+
+
+	public bool InvalidValuesInTemperature() //Due to crappy unity serialisation
```

Comment:
what happened that made you need this?

Reply:
if you add it to a pre-existing class unity defaults the values to 0